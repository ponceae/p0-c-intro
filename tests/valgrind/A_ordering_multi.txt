==3257610== Memcheck, a memory error detector
==3257610== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==3257610== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==3257610== Command: ../intro -g -t 100 -u inputs/A_uniq_nonseq.txt -c inputs/A_noexist.txt -g -t 20 -u inputs/A_uniq_simple.txt -c inputs/B_cat.txt
==3257610== 
../intro: invalid option -- 'u'
Invalid argument.
==3257610== 
==3257610== HEAP SUMMARY:
==3257610==     in use at exit: 0 bytes in 0 blocks
==3257610==   total heap usage: 1 allocs, 1 frees, 8,192 bytes allocated
==3257610== 
==3257610== All heap blocks were freed -- no leaks are possible
==3257610== 
==3257610== For lists of detected and suppressed errors, rerun with: -s
==3257610== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
