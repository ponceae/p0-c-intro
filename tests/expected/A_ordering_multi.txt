Goodbye!
== HEADING ==

This is some sample text. It is in a longer paragraph so that there are multiple
lines of output. After this paragraph, there is single blank line before the
second (and final) paragraph.

This is the final paragraph. There is also a blank line after this sentence.

odd
2
triodd
4
odd
tri
odd
8
triodd
10
odd
tri
odd
14
triodd
16
odd
tri
odd
20
aaa
bbb
ccc
